pkg/report/provider.go

```diff
@@ -0,0 +1,26 @@
+package report
```

Comment:
Is this file used?

Reply:
No. I was not able to clearly understand the dummy-provider reference repo, so I was not sure if I should add this function or not.

Reply:
let's remove this  if it's not used

Reply:
removed